File: SeungMin/황승민:4주차 과제/Q04.c

// 파티가 끝나고 난 뒤 - https://www.acmicpc.net/problem/2845

#include <stdio.h>

int main(void) {

    int L , P, m;
    int arr[5];
    scanf("%d%d", &L, &P);
    m = L * P;

    scanf("%d%d%d%d%d", &arr[0], &arr[1], &arr[2], &arr[3], &arr[4]);

    for (int i = 0; i < 5; i++) {
        printf("%d ", arr[i] - m);
    }

    return 0;
}

File: EunSeo/javaWeek2/Q04.java
package java_week2;

import java.util.Scanner;

public class Q04 {
    // 중앙 이동 알고리즘 - https://www.acmicpc.net/problem/2903
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();
        int answer = 1;

        for (int i = 0 ; i < n ; i++) {
            answer *= 2;
        }

        answer = (answer + 1) * (answer + 1);
        System.out.println(answer);

    }
}